.ci/env/arm-toolchain.cmake

```diff
@@ -0,0 +1,38 @@
+#===============================================================================
+# Copyright contributors to the oneDAL project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+
+# Set the CMake system name to inform CMake that we are cross-compiling
+set(CMAKE_SYSTEM_NAME Linux)
+
+# Set the cross-compilation prefix for the toolchain
+set(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
+set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
+
+# Set the target architecture
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+# Set compiler flags and options for ARMv8-A architecture with SVE
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+sve")
```

Comment:
Should these be set in the CMake toolchain? The march flags are dealt with in the build steps of the various projects, so think that this isn't necessary. If it is necessary to set the march flags, this should be done outside of the toolchain file, though

Reply:
for now only tbb build is dependent on this, no other project is currently using this toolchain.

Reply:
currently only tbb needs the toolchain file and since we wil be compiling only for sve this can be used?